src/mbgl/util/stopwatch.hpp

```diff
@@ -4,9 +4,21 @@
 #include <mbgl/util/chrono.hpp>
 
 #include <string>
+#include <sstream>
 
 namespace mbgl {
 namespace util {
+    
+#ifdef MBGL_TIMING
+#define MBGL_TIMING_START(watch)  std::shared_ptr<util::stopwatch> watch = std::make_unique<util::stopwatch>(Event::Timing);
+#define MBGL_TIMING_FINISH(watch, message) \
```

Comment:
Probably worth considering wrapping these lines in a `do { ... } while (0)`

Reply:
Good point @julianrex . (this is basically to keep the scope of `messageStream` constrained and prevent collisions with code outside the scope)